app/views/admin/navigation_links/index.html.erb

```diff
@@ -17,7 +17,7 @@
               <div class="crayons-icon crayons-icon--default">
                   <%= link[:icon].html_safe %>
               </div>
-              <%= link[:name] %><span id="reading-list-count" class="c-indicator"></span>
+              <%= link[:name] %><span class="reading-list-count c-indicator"></span>
```

Comment:
```suggestion
              <%= link[:name] %><span class="js-reading-list-count c-indicator"></span>
```

(small refactor / blocking) - I know it's such a tiny thing to request, but when we introduce a class only to reference it in JS, we try to prefix with `js-` just so it's clear to future devs looking at it that there's no styling concern here.

(same with the one below)